css-two-column-layout/index.html

```diff
@@ -0,0 +1,101 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column-full">
+        <h1 class="all-cap karla center">scale, speed, agility</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-full heading-gap">
+        <p class="karla center gray-text">We&apos;ve dedicated thousands of man hours to making healthcare institutions in the Helium<br>
+          network run 10x more efficiently and deliver an unparalleled quality of care.
+        </p>
+      </div>
+    </div>
+
+    <div class="row">
+      <div class="column-12pct">
+        <img src="images/software.svg" alt="software">
+      </div>
+      <div class="column-35pct">
+        <div class= "blank-space"></div>
```

Comment:
What is the point of this? If you're using this to push down elements, why not just use `margin` or `padding`?